docs/src/topics/getting-started.md

```diff
@@ -31,6 +31,19 @@ export LINODE_TOKEN=<your linode PAT>
 export LINODE_CONTROL_PLANE_MACHINE_TYPE=g6-standard-2
 export LINODE_MACHINE_TYPE=g6-standard-2
 ```
+## Specifying the region and image
+
+### Image
+The default image is linode/ubuntu22.04 if nothing is specified. This can be overridden, but currently only ubuntu is 
```

Comment:
There should be some other supported versions too and Debian 11:
```
linode-cli images ls | grep cloud-init
│ linode/debian11            │ Debian 11                  │ Debian    │                           │ True      │ 1500  │ available │ cloud-init   │
│ linode/ubuntu20.04         │ Ubuntu 20.04 LTS           │ Ubuntu    │                           │ True      │ 2000  │ available │ cloud-init   │
│ linode/ubuntu22.04         │ Ubuntu 22.04 LTS           │ Ubuntu    │                           │ True      │ 3500  │ available │ cloud-init   │
│ linode/ubuntu23.10         │ Ubuntu 23.10               │ Ubuntu    │                           │ True      │ 3500  │ available │ cloud-init   │
```

Reply:
the issue is our install scripts for kubeadm are ubuntu specific, let me try and rephrase it to say our kubeadm template only supports ubuntu currently

Reply:
I'm not sure what the timeline is on getting the other Ubuntu versions compatible with cloud-init, doesn't seem like we can just say 20.04 and up:
```
linode-cli images ls | grep ubuntu | sort
│ linode/ubuntu16.04lts      │ Ubuntu 16.04 LTS           │ Ubuntu    │                           │ True      │ 2700  │ available │              │
│ linode/ubuntu18.04         │ Ubuntu 18.04 LTS           │ Ubuntu    │                           │ True      │ 2700  │ available │              │
│ linode/ubuntu20.04         │ Ubuntu 20.04 LTS           │ Ubuntu    │                           │ True      │ 2000  │ available │ cloud-init   │
│ linode/ubuntu22.04         │ Ubuntu 22.04 LTS           │ Ubuntu    │                           │ True      │ 3500  │ available │ cloud-init   │
│ linode/ubuntu22.10         │ Ubuntu 22.10               │ Ubuntu    │                           │ True      │ 3500  │ available │              │
│ linode/ubuntu23.04         │ Ubuntu 23.04               │ Ubuntu    │                           │ True      │ 3500  │ available │              │
│ linode/ubuntu23.10         │ Ubuntu 23.10               │ Ubuntu    │                           │ True      │ 3500  │ available │ cloud-init   │
```
Until then we might want to call out specific versions?

Reply:
with the cloud-init shim any version of ubuntu should work, we are just doing some ubuntu specific stuff in our preKubeadm commands so we can't support other OSes for kubeadm